tests/datasets/tasks/indicator_data_extraction/test_extraction.py

```diff
@@ -57,8 +64,10 @@ def test_three_geographies(self, indicator, geography):
             idata = IndicatorData.objects.get(geography=g)
             assert idata.data[0]["geography"] == g.pk
 
-    def test_universe(self, indicator, geography):
-        universe = {"data__gender": "female", "data__age__lt": "17"}
+    def test_universe(self, indicator, geography, universe):
+        # Add universe to indicator
```

Comment:
I think it's worth adding a check here to ensure there is no universe in the indicator before being set

Reply:
updated test to create and use new indicator rather than updating older one